source/components/GlobalNavigation/components/GlobalNavigationNotifications/CardText.js

```diff
@@ -0,0 +1,145 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useTranslation } from 'react-i18next';
+import get from 'lodash/get';
+
+import {
+    isAnnouncement,
+    isDiscussionPostUpvote,
+    isDiscussionReply,
+    isDiscussionReplyUpvote,
+} from '../../models/notificationTypes';
+
+const getReplyMessageBody = (t, { title, totalUniqueActors, latestActors, postTitleMarkup }) => {
```

Comment:
Maybe we could use a slightly more describe name than `t`? "translate" or "i18n"?